src/heads-up/components/properties-item.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="properties-item">
+    <app-tooltip>
+      <span class="properties-item__term">
+        <slot />
+      </span>
+
+      <template v-slot:value>
+        <span class="properties-item__description">
+          <slot name="value" />
+        </span>
+      </template>
+
+      <template v-slot:info>
+        <!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
Maybe just throw out the vue/no-v-html of the whole project? Or is it just a warning?